packages/transform-metering/src/meter.js

```diff
@@ -7,8 +7,8 @@ const { ceil } = Math;
 const ObjectConstructor = Object;
 
 // eslint-disable-next-line no-bitwise
```

Comment:
This eslint-disable comment isn't needed.  Nor is the one on line 1.

Reply:
Oops, missed this. Addressed in #2309